gerrit-cs-status/src/main/java/com/tw/go/plugin/GerritBuildStatusNotifierPlugin.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2019 ThoughtWorks, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.tw.go.plugin;
+
+import com.thoughtworks.go.plugin.api.annotation.Extension;
+import com.tw.go.plugin.provider.Provider;
+
+import java.lang.reflect.Constructor;
+import java.util.Properties;
+
+@Extension
+public class GerritBuildStatusNotifierPlugin extends BuildStatusNotifierPlugin {
+    @Override
+    protected Provider loadProvider() {
+        try {
+            Properties properties = new Properties();
+            properties.load(getClass().getResourceAsStream("/defaults.properties"));
+            Class<?> providerClass = Class.forName(properties.getProperty("provider"));
```

Comment:
Return the provider object using `new` instead of using `Class.forName(...)` as we have only one provider per module.

Reply:
Done.